src/System.Management.Automation/engine/CommandCompletion/CommandCompletion.cs

```diff
@@ -643,7 +643,7 @@ private static List<CompletionResult> InvokeLegacyTabExpansion(PowerShell powers
             char quote;
             var lastword = LastWordFinder.FindLastWord(legacyInput, out replacementIndex, out quote);
             replacementLength = legacyInput.Length - replacementIndex;
-            var helper = new CompletionExecutionHelper(powershell);
+            var helper = new PowerShellExecutionHelper(powershell);
```

Comment:
This replacement is reasonable from the commits history point of view, but in the final diff, I don't see why do we want to remove one class/file and replace it with another one with the similar duties. Maybe we should keep the same name, so the caller doesn't need to change?

It will also help, if PowerShell team will decide to bring some changes back to windows, for that see https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/map-json-files.md